react/features/invite/functions.js

```diff
@@ -266,7 +266,7 @@ export function getInviteTypeCounts(inviteItems: Array<Object> = []) {
  */
 export function invitePeopleAndChatRooms( // eslint-disable-line max-params
         inviteServiceUrl: string,
-        inviteUrl: string,
+        inviteUrl: ?string,
```

Comment:
how can this be optional? where would we be inviting users? I think the fix must be on the caller side, to make sure we always pass a string

Reply:
Probably, but then we need to change the inviteUrl generation logic, because that can explicitly return undefined (I guess there was a reason for it)

Reply:
Then let's change that, as it is this change just hides the problem.